libs/map/src/components/search/vl-map-search.ts

```diff
@@ -95,20 +96,26 @@ export class VlMapSearch extends BaseElementOfType(HTMLElement) {
         });
     }
 
-    _addSelectChangeListener() {
+    private addSelectChangeListener() {
         this._selectElement.addEventListener('change', (e) => {
             if (e.target.location) {
                 e.target.location.then((location) => {
                     if (this._onSelect) {
                         this._onSelect(location);
                     } else {
-                        this._zoomTo(location);
+                        this.zoomTo(location);
                     }
                 });
             }
         });
     }
 
+    private preventKeypressPropagation() {
+        this.addEventListener('keypress', (e) => {
+            e.stopPropagation();
```

Comment:
Zeker dat we alle keypresses moeten stoppen? Zijn er geen andere shortcuts die we willen behouden?

Reply:
Zonder preventDefault werken browser shortcuts nog. Of bedoel je als een parent element van de vl-map-search een bepaalde shortcut heeft geïmplementeerd?